spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -23,13 +23,67 @@
 
 The MicroProfile pom will identify the contents of each MicroProfile release.
 The contents will be those features and versions agreed to by the MicroProfile community.
-These features may be external to the MicroProfile community (Java EE features such as JAX-RS 2.1), or they be a component of the MicroProfile project (such as Config 1.3).
+These features may be external to the MicroProfile community (Jakarta EE features such as
+Jakarta RESTful Web Services 2.1),
+or they be a component of the MicroProfile project (such as Config 2.0).
 
 Since the MicroProfile repo is an "umbrella" project, there will be no individual API or TCK generated for this component.
 Only this Specification will be generated and maintained to document the contents of each release.
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile4.0-M1]]
+=== MicroProfile 4.0 Milestone 1 (Q32020)
+// MicroProfile 4.0-M1 is the 14th platform release for the
+// https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+MicroProfile 4.0-M1 is based on Jakarta EE 8, the first MicroProfile release to
+be based on Jakarta EE (replacing the role of Java EE).
+Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with
+Jakarta EE dependencies for all MicroProfile specifications, so all MicroProfile specifications
+are updated to major versions.
+
+The following specifications include API breaking changes:
+
+* https://github.com/eclipse/microprofile-config/issues?q=is%3Aissue+milestone%3A%22MP+Config+2.0%22+label%3A%22incompatible+change+%3Awarning%3A%22+is%3Aclosed[Config]
+* https://github.com/eclipse/microprofile-fault-tolerance/issues?q=is%3Aissue+milestone%3A3.0+is%3Aclosed+label%3A%22Incompatible+Changes%22[Fault Tolerance]
+* https://github.com/eclipse/microprofile-health/issues?q=is%3Aissue+label%3Abreaking_change+is%3Aclosed[Health]
```

Comment:
https://github.com/eclipse/microprofile-health/issues/271 raised as it does not contain the Incompatible changes section.
https://download.eclipse.org/microprofile/microprofile-health-3.0-RC3/microprofile-health-spec.html#release_notes_3_0

Reply:
@antoinesd chose to just put them in bold -> "Items in bold are breaking changes." but they are there. Is that a problem?

Reply:
Personally I prefer they are nested bullets, but I am not going to die on the hill for this one :-)  I do like the idea of a consistent approach across specs, though.

Reply:
@Emily-Jiang, will fix.

Reply:
so should we fix it and do another RC?

Reply:
@jclingan @Emily-Jiang can anyone answer the above question, please? ^^

Reply:
That's fine for now, @xstefank.